internal/command/auth/login.go

```diff
@@ -40,17 +43,42 @@ func Login(run types.RunFunc) *cobra.Command {
 				return fmt.Errorf("failed to save token: %w", err)
 			}
 
+			// update the context with the new token
+			ctx = auth.TokenToContext(ctx, token)
+
 			fmt.Println("Login succeeded")
 
-			err = config.Save(ctx, &config.Config{})
+			cnf := &config.Config{}
+
+			http := client.New(ctx, apiURL)
+			organizations, err := organization.List(ctx, http)
 			if err != nil {
-				return fmt.Errorf("failed to save configuration: %w", err)
+				return fmt.Errorf("failed to list organizations: %w", err)
+			}
+
+			fmt.Println()
+
+			if len(organizations) == 1 {
+				cnf.Organization = organizations[0].ID
+
+				fmt.Println("Automatically selected the only organization you have access to: " + organizations[0].ID)
+			} else {
+				fmt.Println(
+					"You do not have an organization selected, select one using:\n" +
+						"\n" +
+						"\tjsctl config set organization [name]\n" +
+						"\n" +
+						"You have access to the following organizations (run 'jsctl organizations list'):",
+				)
+
+				for _, org := range organizations {
+					fmt.Println("  - " + org.ID)
+				}
 			}
 
-			cnf, ok := config.FromContext(ctx)
-			if !ok || cnf.Organization == "" {
-				fmt.Println("You do not have an organization selected, select one using: \n\n\tjsctl config set organization [name]\n\n" +
-					"To view organizations you have access to, list them using: \n\n\tjsctl organizations list")
+			// save the configuration to disk
+			if err = config.Save(ctx, cnf); err != nil {
+				return fmt.Errorf("failed to save configuration: %w", err)
```

Comment:
The problem with this is that a user with multiple organizations must always select their organization after logging in.
Which is inconvenient.

In the current code, the previously selected organization would be remembered.

Isn't it important to load the `config.FromContext` anyway, to ensure that we don't clobber other settings that the user may have configured.

Reply:
we already had this unconditional save, so I don't think that this PR introduces new unexpected behaviour.
https://github.com/jetstack/jsctl/pull/91/files#diff-05450dabe1bdf8a081ec573326786d00b4afdda95ed2a2c725e985eb217e6b80L45

Reply:
You are right, I updated the code.